### PR TITLE
docs: first pass at a guide for new contributors

### PR DESCRIPTION
**Pull request by espy**

### CODE REVIEW

File: NEW-CONTRIBUTORS.MD [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+While this _is_ cryptography-heavy JavaScript software, and those skills are especially relevant, an Open Source project such as this one has a wide range of needs, and there are a lot of opportunities to help:
+
+- There are usually some non-cryptography related JS (and TS!) tasks
+- You can expand or improve the test suite
+- You can take the project for a test drive and update the dev setup documentation for your platform (see [CONTRIBUTING.md](/CONTRIBUTING.md))
+- You can improve the various documentation resources in general, for example by looking for [documentation issues](https://github.com/openpgpjs/openpgpjs/issues?q=is%3Aissue+is%3Aopen+label%3ADocumentation), or just picking something to expand upon, edit or check for typos
+- You can improve the build process
+- You can do manual tests and UI/UX evaluations of features or tools and help improve their usability
+- You can improve or maintain [our website](https://openpgpjs.org/)
+- You can evangelize for the project: present or talk about OpenPGP.js in your company, local tech meetup, youtube channel, or similar circles
```
Comment (AlbaHerrerias) [inline NEW-CONTRIBUTORS.MD:42]: This is great! I have something following the same idea here https://github.com/neighbourhoodie/openpgpjs/pull/5/files#diff-eca12c0a30e25b4b46522ebf89465a03ba72a03f540796c979137931d8f92055R187

Should we merge them?

Comment (espy) [reply NEW-CONTRIBUTORS.MD:42]: Yeah, I was wondering where to best put this. I think it might be more relevant to new contributors? Really not sure.

Comment (AlbaHerrerias) [reply NEW-CONTRIBUTORS.MD:42]: I agree. Let's put it here, we can always ask them for their preference

**Review by AlbaHerrerias — APPROVED** [review]

File: NEW-CONTRIBUTORS.MD (hunk 1 above)
Comment (AlbaHerrerias) [inline NEW-CONTRIBUTORS.MD:42]:
```suggestion
- You can evangelize for the project: present or talk about OpenPGP.js in your company, local tech meetup, youtube channel, or similar circles
- You can answer questions on our [Gitter](https://gitter.im/openpgpjs/openpgpjs) or our [GitHub discussions](https://github.com/openpgpjs/openpgpjs/discussions)
```

**Pull request merged**